src/snowflake/snowpark/dataframe_analytics_functions.py

```diff
@@ -21,7 +21,7 @@
 from snowflake.snowpark.window import Window
 
 # "s" (seconds), "m" (minutes), "h" (hours), "d" (days), "w" (weeks), "t" (months), "y" (years)
-SUPPORTED_TIME_UNITS = ["s", "m", "h", "d", "w", "t", "y"]
+SUPPORTED_TIME_UNITS = ["s", "m", "h", "d", "w", "mm", "y"]
```

Comment:
will this change break older customers as "t" is no longer supported?
also, nit: update comment above

Reply:
I think you are looking into the wrong diff. This line was added in this PR.

Reply:
yeah. I got redirected to single commit diff.